charts/tidb-operator/values.yaml

```diff
@@ -163,6 +163,10 @@ scheduler:
   # Refer to https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
   podAnnotations: {}
 
+  # additional annotations for the configmap, mainly to prevent spinnaker versioning the cm
+  configmap:
+    annotations: {}
```

Comment:
how about renaming it to something like `configmapAnnotations` just as `podAnnotations` did?

Reply:
thanks for pointing out! fixed